changelog/_unreleased/2020-09-22-fix-country-sorting-in-cart.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Fix country sorting in cart
+issue: NA
+author: Huzaifa Mustafa
+author_email: 24492269+zaifastafa@users.noreply.github.com 
+author_github: zaifastafa
+---
+# Core
+*  Added criteria in `\Shopware\Storefront\Page\Checkout\Cart\CheckoutCartPageLoader::getCountries` to first sort by 
```

Comment:
changelog does not fit the changes anymore 😉

Reply:
Hey @zaifastafa, ccould you please update the changelog? Afterwards i can import your changes and put them trough our internal review to be merged (hopefully) soon.